.travis.yml

```diff
@@ -24,6 +24,11 @@ install:
   - pip install -r ../requirements.txt
   - sudo apt-get install coreutils libxml2-utils
 script:
+  - echo "COMPOSER_DAG_BUCKET" > .COMPOSER_DAG_BUCKET
```

Comment:
I think we should read those from environment values if present. Then we will be able to configure environment values in Travis and run the system tests there automatically (that's an ultimate goal.